tests/test_channel.py

```diff
@@ -149,3 +149,235 @@ async def test_channel_beforeAndAfterExecutionFinished(websocket, context_id):
+                    channel_1('CHANNEL_1_MESSAGE');
+                    channel_2('CHANNEL_2_MESSAGE');
+                }""",
+                "arguments": [{
+                    "type": "channel",
+                    "value": {
+                        "channel": "CHANNEL_1",
+                        "ownership": "root",
+                    },
+                }, {
+                    "type": "channel",
+                    "value": {
+                        "channel": "CHANNEL_2",
+                        "ownership": "root",
+                    },
+                }],
+                "target": {
+                    "context": context_id
+                },
+                "awaitPromise": False,
+                "resultOwnership": "root"
+            }
+        })
+
+    resp = await read_JSON_message(websocket)
+    assert resp == {
+        "method": "script.message",
+        "params": {
+            "channel": "CHANNEL_1",
+            "data": {
+                "type": "string",
+                "value": "CHANNEL_1_MESSAGE"
+            },
+            "source": {
+                "context": context_id,
+                "realm": ANY_STR,
+            }
+        }
+    }
+    resp = await read_JSON_message(websocket)
+    assert resp == {
+        "method": "script.message",
+        "params": {
+            "channel": "CHANNEL_2",
+            "data": {
+                "type": "string",
+                "value": "CHANNEL_2_MESSAGE"
+            },
+            "source": {
+                "context": context_id,
+                "realm": ANY_STR,
+            }
+        }
+    }
+
+
+@pytest.mark.parametrize(
+    "serialization_options, expected_result",
+    [
+        ({}, {
+            "type": "object",
+            "value": [[
+                "a", {
+                    "type": "node",
+                    "value": {
+                        "nodeType": 1,
+                        "childNodeCount": 0,
+                        "shadowRoot": None,
+                        "localName": "body",
+                        "namespaceURI": "http://www.w3.org/1999/xhtml",
+                        "attributes": {}
+                    },
+                    "sharedId": ANY_SHARED_ID
+                }
+            ]],
+            "handle": ANY_STR
+        }),
+        ({
+            "maxDomDepth": 0
+        }, {
+            "type": "object",
+            "value": [[
+                "a", {
+                    "type": "node",
+                    "value": {
+                        "nodeType": 1,
+                        "childNodeCount": 0,
+                        "shadowRoot": None,
+                        "localName": "body",
+                        "namespaceURI": "http://www.w3.org/1999/xhtml",
+                        "attributes": {}
+                    },
+                    "sharedId": ANY_SHARED_ID
+                }
+            ]],
+            "handle": ANY_STR
+        }),
+        # ({
```

Comment:
Could you remove these comments? Alternatively, if they are meant to exist, then keep them but skip the tests with `pytest.xfail(reason="TODO: fail")`

Reply:
done